rialto_airflow/publish/publication.py

```diff
@@ -399,6 +401,50 @@ def _openalex_end_page(openalex_json: dict) -> str | None:
     return None
 
 
+def _volume(row) -> str | None:
+    vol = first(
+        row,
+        rules=[
+            JsonPathRule("openalex_json", "biblio.volume"),
+            JsonPathRule("dim_json", "volume"),
+            JsonPathRule(
+                "pubmed_json", "MedlineCitation.Article.Journal.JournalIssue.Volume"
+            ),
+            JsonPathRule("sulpub_json", "journal.volume"),
+        ],
+    )
+
```

Comment:
The type signature for `first()` has evolved to being quite loose, and now as a result usage requires a bit of additional checking to satisfy mypy.